cross/sshfs-fuse/PLIST

```diff
@@ -0,0 +1,83 @@
+bin:bin/sshfs
+bin:bin/fusermount
+bin:bin/gdbus
+bin:bin/gdbus-codegen
+bin:bin/gio-querymodules
+bin:bin/glib-compile-resources
```

Comment:
This PLIST needs work ;) It currently contains stuff that is created by glib, libffi etc. 
Remove everything _unless created directly_ by `sshfs-fuse`. Usually, the sshfs-fuse.plist file is a good starting point, but it will probably show entries that are not already in other plists.